app/views/conversations/_messages.haml

```diff
@@ -6,17 +6,12 @@
       .media-left
         = owner_image_tag(:thumb_small)
       .media-body
-        = form_for [conversation, Message.new], html: {class: "control-group"} do |message|
+        = form_for [conversation, Message.new], html: {id: "response-message", class: "control-group"} do |message|
           .form-group
-            %label#messageLabel.sr-only{for: "message_text"}
-              = t("conversations.new.message")
-            = message.text_area :text,
-                                rows:     5,
-                                tabindex: 1,
-                                class:    "form-control form-group",
-                                aria:     {labelledby: "messageLabel"}
+            %label.sr-only#message-label{for: "response-message-text"}= t("conversations.new.message")
+            = message.text_area :text, rows: 5, tabindex: 1, id: "response-message-text",
+                class: "form-control form-group conversation-message-text", aria: {labelledby: "message-label"}
```

Comment:
At least for me the readability was better before when we had one attribute per line.